protocol/testing/e2e/gov/bridge_test.go

```diff
@@ -101,7 +118,43 @@ func TestUpdateProposeParams(t *testing.T) {
 			},
 			expectedProposalStatus: govtypesv1.ProposalStatus_PROPOSAL_STATUS_PASSED,
 		},
-		"Fail: invalid authority": {
+		"Failure: negative propose delay duration": {
+			msg: &bridgetypes.MsgUpdateProposeParams{
+				Authority: authtypes.NewModuleAddress(govtypes.ModuleName).String(),
```

Comment:
I believe there is a constant somewhere you can use for this these days... `lib.GovModuleAddress.String()`

Reply:
Can you propagate here and below?